### ISSUE TITLE
`aria-label` and `a11yTitle` research

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6433]
A discussion came up about why we allow both `aria-label` and `a11yTitle` to be passed to components. Right now to my knowledge they are being used interchangeably on components. We should spend some time looking into this area to determine if we should keep `a11yTitle` as is, if it should be deprecated in favor of `aria-label`, or if `aria-label` and `a11yTitle` should have different behavior (for example `aria-label` is only used for the actual aria-label on the component and not anywhere else and `a11yTitle` is used for other accessibility messages).

Deliverables: 
- [ ] Collect cases where aria-label and a11yTitle are used for anything besides the aria-label in a component
- [ ] Define desired behavior
- [ ] If the desired direction requires changes in v3 add notes to the v3 wiki page
